Sources/StreamChatSwiftUI/Resources/en.lproj/Localizable.strings

```diff
@@ -148,7 +148,7 @@
 "composer.polls.duplicate-option" = "This is already an option";
 "composer.polls.multiple-answers" = "Multiple answers";
 "composer.polls.suggest-option" = "Suggest an option";
-"composer.polls.type-number-from-1-and-10" = "Type a number from 1 and 10";
+"composer.polls.type-number-from-2-and-10" = "Type a number from 2 and 10";
```

Comment:
This is a breaking change, but not a big deal. Either way, we should not have hard coded numbers in he localize keys. Lets change to something like this:

```suggestion
"composer.polls.number-min-max-range" = "Type a number from 2 and 10";
```

Reply:
Updated 👍